### DISCUSSION TITLE
Keycloak works on internal URL (ok) - public URL hangs

### DISCUSSION THREAD

**Discussion by ejbest** [keycloak/keycloak#30774] [Q&A]
Both Public / Internal URLs are behind the same reverse proxy 

Is there a variable or otherwise Keycloak setting that is needed for the Public Inbound URL (as it is clearly connected but is hanging with a spinning message).

https://internalurl/keycloak/realms/ABC/account/#/       
WORKS GREAT !! 

https://www1.xyz.com/keycloak/realms/ABC/account/#/ 
HANGS WITH SPINNING MESSAGE 

-  Same Reverse Proxy & Routing and DNS
- Is there a setting in Keycloak that would "bless the url https://www1.xyz.com/keycloak/"?
- Have set "frontendUrl" and "hostname" to this value :"https://www1.xyz.com/keycloak/"?

Any Guidance is Appreciated 
**Thank you in Advance**

**Comment by codespearhead** [comment]
Try what's suggested in [this answer](https://github.com/keycloak/keycloak/discussions/28279#discussioncomment-9001818).

Potential duplicates: https://github.com/keycloak/keycloak/discussions/28279 , https://github.com/keycloak/keycloak/discussions/20098
